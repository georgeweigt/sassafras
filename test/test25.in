data wine ;
input color $ x1 x2 x3 x4 x5 x6 x7 x8 x9 x10 x11 y ;
infile "wine.txt" ;
* y squared ;
y2=y**2 ;
proc means n mean clm maxdec=6 ;
var y y2 ;
class color ;
run ;
