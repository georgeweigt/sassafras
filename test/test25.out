   COLOR     Variable        N          Mean     95% CLM MIN     95% CLM MAX
   red       Y            1599      5.636023        5.596410        5.675635
             Y2           1599     32.416510       31.960513       32.872508
   white     Y            4898      5.877909        5.853101        5.902718
             Y2           4898     35.334014       35.037160       35.630868

